### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by xiaopanchen** [kyungwon213/AD_ViT#1]
I have read your paper "Attribute De-biased Vision Transformer (AD_ViT) for Long-Term Person Re-identification". Your idea is very great. Now, I am reading your code. But, I cannot find the attribute file. For example,  the code "with open(os.path.join(self.dataset_dir, 'PAR', 'baseline', cfg.MODEL.PAR_MODEL, cfg.MODEL.PAR_DATASET, list_path), 'r') as f:" in ltcc.py. I know that the attribute predictions should be obtained by pre-training.  Can you share the code of the APR network you used. Without the attribute files, I cannot run your code.
Furthermore, there are not MSLoss in your code. Can you provide this loss?
Thank you very much.

**Comment by kyungwon213** [comment]
Hi, sorry for the late reply and thank you for your interest. 
Please download the prediction values in this link: https://drive.google.com/drive/folders/1xRKLhjZC7d2vVJ68-J1DfzWq5iv7Spf2?usp=share_link. All the losses are defined in the make_loss.py file in the loss folder. Please let me know if you have any other questions.

**Comment by xiaopanchen** [comment]
THk
